include/hnswlib/hnswalg.h

```diff
@@ -1191,6 +1191,34 @@ namespace hnswlib {
             return cur_c;
         };
 
+        std::priority_queue<std::pair<dist_t, labeltype >>
+        searchSubset(const void *query_data, const void *subset_data, size_t k, size_t n) const {
+            std::priority_queue<std::pair<dist_t, labeltype >> result;
+            if (cur_element_count == 0) return result;
+
+            labeltype *candidate_id = (labeltype *) subset_data;
+
+            for (size_t i = 0; i < std::min(n, k); i++) {
+                dist_t dist = fstdistfunc_(query_data, getDataByLabel(*candidate_id), dist_func_param_);
+                result.push(std::pair<dist_t, labeltype>(dist, *candidate_id));
+                candidate_id++;
+            }
+
+            dist_t lastdist = result.top().first;
+
+            for (size_t i = k; i < n; i++) {
```

Comment:
it seems weird that u are checking before the `min` between n and k and now u are assuming one is smaller than the other?

Can we use better expressive names other than `k` and `n`?

Reply:
> now u are assuming one is smaller than the other?

No assumption here. If `n < k`, the for-loop above can also work correctly.

Reply:
u mean, it will not enter.

It would be useful to understand what is the meaning of these variables and have more meaningful names